Dockerfile

```diff
@@ -1,66 +1,58 @@
-############################################################
-## Dockerfile to build the ga4gh server on mod_wsgi-express
-## Configurable to use a local dataset
-############################################################
-FROM ubuntu
+# Using multi stage to prevent keeping a second copy of the package in the image
+FROM centos:7.6.1810
+RUN yum -y install epel-release
+RUN yum -y install python36-pip.noarch \
+ git \
+ libffi-devel.x86_64 gcc-c++.x86_64 \
+ python36-devel.x86_64 openssl-devel \
+ libxml2-devel.x86_64 libxslt-devel.x86_64  libcurl-devel.x86_64 make gcc  \
+ && pip3 install --upgrade pip setuptools
 
-# Originally created by Steve Hershman GitHub @hershman
-# previously maintained by Alastair Firth, and Maciek Smuga-Otto of the
-# UCSC Genomics Institute
-MAINTAINER David Steinberg <david@resium.com>
+ENV SCHEMA_V=v1.0.0 INGEST_V=v1.3.0
 
-# Update the sources list
-RUN apt-get update  --fix-missing
-RUN apt-get upgrade --yes
+RUN  pip install \
+  git+https://github.com/CanDIG/candig-schemas.git@${SHEMA_V}#egg=candig_schemas  \
```

Comment:
should we use the pypi released version?

Reply:
There's a few options here.  I'd be inclined to say that the Dockerfile in GitHub should probably just use the current master branch of all of the tools, and that when we push releases to pypi we should make sure appropriately tagged docker images are published to quay.io or dockerhub, whichever we decide to use.  I'm looking around now to see what other projects do.

Reply:
Using only released version of the other tools is the way to go. I would use the tip of master if and only if we only have the latest stable release there. I do not want unfiltered bugs from other tool to be include in the dockerfile that will be used to do releases.

Reply:
I think people who are developing in this repo and so using this Dockerfile (which just uses local code, not a release of candig-server) to test are likely also making changes in -schemas and/or -ingest, so pinning to releases won't make sense.  (The fact that they're developed together so often means that breaking up the repos that way doesn't really make sense, but that's what we've inherited and until we start splitting things up and rearranging them, we're stuck with it).